src/main/java/com/my/relink/controller/TradeController.java

```diff
@@ -0,0 +1,25 @@
+package com.my.relink.controller;
+
+import com.my.relink.domain.trade.dto.TradeRequestDto;
+import com.my.relink.domain.trade.dto.TradeRequestResponseDto;
+import com.my.relink.service.TradeService;
+import com.my.relink.util.api.ApiResult;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TradeController {
+    private final TradeService tradeService;
+
+    public TradeController(TradeService tradeService) {this.tradeService = tradeService;}
+
+    @PostMapping("/trades/{tradeId}/request")
+    public ResponseEntity<ApiResult<TradeRequestResponseDto>> requestTrade(@PathVariable(name = "tradeId") Long tradeId, @RequestBody TradeRequestDto tradeRequestDto) {//추후 로그인 유저로 바뀔 얘정
+        TradeRequestResponseDto responseDto = tradeService.requestTrade(tradeId, tradeRequestDto.getUserId());
+        return new ResponseEntity<>(ApiResult.success(responseDto), HttpStatus.CREATED);
```

Comment:
응답으로 거래 id만 반환하는데 굳이 respDto를 거치는 것보다는 직접 반환하는 게 좀 더 좋지 않을까 싶습니다.
```suggestion
    public ResponseEntity<ApiResult<Long>> requestTrade(@PathVariable(name = "tradeId") Long tradeId, @RequestBody TradeRequestDto tradeRequestDto) {//추후 로그인 유저로 바뀔 얘정
        tradeService.requestTrade(tradeId, tradeRequestDto.getUserId());
        return new ResponseEntity<>(ApiResult.success(tradeId), HttpStatus.CREATED);
```

Reply:
아하 그렇군요 수정하도록 하겠습니다!